proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -87,6 +87,25 @@ message MonthlyUsage {
   uint32 total_method_types = 5;
 }
 
+// RepositoryUsage contains the total amount of message, enum and method types
+// for an repository within an organization.
+message RepositoryUsage {
+  // The id of the repository.
+  string repository_id = 1;
+  // The name of the repository.
+  string repository_name = 2;
+  // The id of the repository owner.
+  string owner_id = 3;
+  // // The name of the respository owner.
+  string owner_name = 4;
+  // The total amount of message types for the month and year.
```

Comment:
```suggestion
  // The total amount of message types.
```
I don't think there is any month/year relationship here

Reply:
right good catch, removed